### PR TITLE
Enable AppVeyor CI

### LINKED ISSUE #448: Add a Windows-specific CI such as AppVeyor
We currently utilize Travis as our sole CI, which does testing on Linux for our PRs.

We should consider adding AppVeyor as a second CI so that we can run the tests on Windows too, to be assured that our code runs on Windows without any issues as well (since it is possible that the same code might behave differently on different OS).

### PR DESCRIPTION
**Pull request by yash-chowdhary**
**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [x] Other, please explain: Add AppVeyor CI config file

Fixes #448 

**What is the rationale for this request?**
Ensuring that the code runs on Windows without any issues.

**What changes did you make? (Give an overview)**
Add an `appveyor.yml` config file

**Next steps:**

- [ ] Grant Organisation Access to AppVeyor 
- [ ] Add the build status badge in `README.md` 

**Proposed commit message: (wrap lines at 72 characters)**
Add AppVeyor CI config file

### CODE REVIEW

**Comment by yash-chowdhary** [comment]
Ready for review

**Comment by yash-chowdhary** [comment]
@yamgent can I get your review on this?

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# AppVeyor configuration file
+environment:
+  nodejs_version: "8"
```
[the remark this answers is not in the document]
Comment (acjh) [reply appveyor.yml]: Aside: we should update the developer guide to recommend v10.13.0 (LTS) or higher, as we did for the user guide in #953.

[the remark this answers is not in the document]
Comment (yash-chowdhary) [reply appveyor.yml]: > Node.js v8 reached End-of-life on 2019-12-31

We should change the version in our `travis.yml` file too

**Pull request merged**